setup.py

```diff
@@ -56,7 +56,7 @@ def read(filename):
         "scipy>=1.1, <2",
         "matplotlib>=2.2",
         "tifffile>=2020.9.30",
-        "tabulate==0.8.6",
+        "tabulate>=0.8.8, <0.8.10",
```

Comment:
Why specifically `<0.8.10`? Usually, the top is set at a semver breaking change boundary, e.g. `<0.9`.

Reply:
Should indeed just have gone with `<0.9`.